privacy-policy.html

```diff
@@ -0,0 +1,278 @@
+        <p class="mt-6 text-base text-black">
+          Cookies are small text files that are placed on your computer by
+          websites that you visit. Our site and our partners use cookies to
+          enhance your experience, analyse traffic and for ad personalisation
+          and measurement. You can restrict or block cookies using your browser
+          settings. Information on how to do this is detailed below in the “How
+          to manage cookies” section. We also use tracking pixels in our
+          marketing emails. These pixels tell us if, and when, you opened an
+          email from us. We use this information to better understand how
+          customers engage with our content and to analyse the effectiveness of
+          our marketing campaigns. You can disable tracking pixels by changing
+          your email settings to block the loading or display of external
+          images.
+        </p>
+        <p class="mt-6 text-xl text-black">How to manage cookies</p>
+        <p class="mt-6 text-base text-black">
+          Some cookies are strictly necessary to allow you to use the website
+          and its features, such as storing items in your basket. Without these
+          cookies these features cannot be provided which may impair the
+          functionality of the website meaning you are unable to shop on our
+          website. Important information about cookies:
+        </p>
+        <p class="mt-4 text-base text-black">
+          <li>
+            You can select the cookies you would like this website to store on
+            your device by changing your browser settings. Each browser allows
+            you to restrict or block cookies in different ways.
+          </li>
+          <li>
+            Details on how to do this in each browser can be found on the ICO
+            website (please note that we are not responsible for the content of
+            external websites):
+            <a
+              href="https://ico.org.uk/your-data-matters/online/cookies/"
+              target="_blank"
+              class="blue-link"
+              >https://ico.org.uk/your-data-matters/online/cookies/
+            </a>
+          </li>
+          <li>
+            Within most browsers you can utilise private browsing modes that
+            typically do not save local data associated with the session, such
+            as cookies. These are typically deleted when you close all the
+            windows.
+          </li>
+          <li>
+            You can install a browser add-on to opt-out of Google Analytics in
+            your browser across all websites (please note that we are not
+            responsible for the content of external websites), visit:
+            <a
+              href="https://tools.google.com/dlpage/gaoptout"
+              target="_blank"
+              class="blue-link"
+              >https://tools.google.com/dlpage/gaoptout
+            </a>
```

Comment:
Add the travel orange colour on the link
Also we want to have a hover state for underline and text-orange-600
Refer to Tailwind Docs to check how to add pseudo classes to your class

Reply:
added  text-travel-orange hover:underline to class